### ISSUE TITLE
Bot will not get past initialization 2/3

### ISSUE THREAD

**Issue by azurinio** [Masuzu/ZooeyBot#1300]
The bot will launch and tell me to open dev tools after launching the chrome extension. The bot then freezes until I close out of devtools and provides me with no 3/3 intialization but instead tracking each page I go to (images attatched). I have tried every which way of launching, i/e have devtools open before launching extension, after launching extension, admin privileges on bot, using chromium instead of chrome, etc. I have even tried messing with the .ini config file to no avail. Any help is appreciated but I've lost 2 days of license to not being able to use it.

**Screenshots**

[https://imgur.com/a/a8do977](url)

I've tried using discord to get any help but the only help I've received so far has not worked because it is basically "hit f12??"

I am user bonelessnuggies#4556 in the Discord server for quicker responses.

**Comment by Mewnya** [comment]
That's actually the other way around. You first need to open the devtools window (F12) before clicking on the "Start" button.

**Comment by azurinio** [comment]
Same result all ways attempted (already tried that but tried again on suggestion)

**Comment by Mewnya** [comment]
I am afraid I can't see this happening unless your firewall blocked the ZooeyBot executable file:
https://streamable.com/ow4u23

**Comment by azurinio** [comment]
Issue has been resolved but now another issue has come up in it's place. Initializaion 3/3 happens and I'm prompted to navigate to the Quest screen. Upon getting to the Quest screen the command prompt closes without any errors.

**Comment by Mewnya** [comment]
This looks very similar to a case reported on [Discord](https://discord.gg/uaJZUJ2) today at 10:55 GMT+0, which seemingly was due to the browser cache.

**Comment by azurinio** [comment]
I just tried it with a clean state and did as follows:

1. Clear all browsing history from Chrome
2. Redownload Zooeybot 3.16.1 and extract into it's own folder. (change jp to en in .ini)
3. Relogin to GBF
4. Wait on Home Screen
5. Reload Zooeybot Chrome Extension
6. while on GBF Home Screen, opening Dev Tools (F12) and launching zooey bot.
7. Redownload key.txt from website.
8. Start Zooeybot again
9. Start Zooeybot extension.
10. Bot gets stuck on 2/3 again.
11. On retry of steps 6-9 the bot gets past 2/3 and succeeds in 3/3 while telling me to navigate to Quest menu.
12. On clicking Quest menu the bot closes.
13. If I loop back and retry all the steps labeled (clearing cache etc,) here it produces the exact same series of events. 

Hopefully that can help at all. I don't see Zooeybot creating any logs or anything that may be of use but I can upload my ini albeit it is the default ini packed with the Zooeybot download asides from changing the locale from jp to en.

**Comment by Mewnya** [comment]
As much I would like to help, this seems like the first case of the sort, too isolated unfortunately.
I would suggest requesting a refund.
